obspy/io/gse2/tests/test_bulletin.py

```diff
@@ -323,6 +357,8 @@ def test_amplitude(self):
         catalog = _read_gse2(filename)
         self.assertEqual(len(catalog), 1)
         amplitudes = catalog[0].amplitudes
+        # test a new feature: don't store an object amplitude if the magnitude
+        # type is not defined
```

Comment:
Is there anything missing here, @thefroid? I see this comment but the test isn't changed..